webui/src/components/editrecord.jsx

```diff
@@ -373,6 +379,22 @@ const EditRecord = React.createClass({
             borderRadius:'4px',
         };
         const isError = this.state.errors.hasOwnProperty(fieldID);
+        if (fieldID == "language_code") {
+             return (
+                <div className="form-group row" key={fieldID} style={{marginTop:'1em'}} title={fieldSchema.get('description')}>
+                    <label htmlFor={fieldID} className="col-sm-3 control-label" style={{fontWeight:'bold'}}>
+                        <span style={{float:'right', color:isError?'red':'black'}}>
+                            {fieldSchema.get('title') || fieldID} {type.required ? "*":""}
+                        </span>
+                    </label>
+                    <div className="col-sm-9" >
+                        <SelectLanguage onSelectLanguage={this.handleLanguageCode} defValue={getValue()} />
```

Comment:
Try using `onSelectLanguage={setValue}`. setValue is defined above and sets the language code directly in the record.